### ISSUE TITLE
Suggestion: Allow changing instruments (program changes) from the QT GUI

### ISSUE THREAD

**Issue by wyatt8740** [munt/munt#70]
Hi.

I don't have a problem with the way Munt currently handles things, since I'm well versed in MIDI by this point, am able to roll my own tools, and use ALSA which gives a decent amount of flexibility and control, but I have a Windows-using friend who might benefit from this.

He makes music, and took interest in my (real) MT-32 when I showed it to him. He wants to use Munt to make some new music of his own. He has a MIDI controller keyboard, but it doesn't provide a button or means to do program changes on it, so his first question to me was "how do I change the instruments?"

So I had to do this nasty hack with VMPK (which only uses General Midi instrument names unless you edit config files) to get him a way to manipulate instruments on Windows, which he promptly forgot how to do.

So my suggestion is relatively straightforward. In that 'drop-down' area that contains the channel statuses and partial states, instead of simple labels naming the channels, maybe there could be `QComboBox`es, each containing all of the preset instruments on each channel.

Perhaps stored instruments could be shown in the boxes as well - but I think just exposing the presets would be enough, since that's all that the LCD and controls on the front panel of an MT-32 can do. If a custom instrument is set, a new item with that instrument name could be added at the top or bottom of the combobox, which self-destructs (disappears) if another preset is selected.

In short, since the real MT-32 has functionality to (awkwardly) set instruments from the front panel, I think it'd be nice if Munt could do so as well in the GUI.

**Comment by sergm** [comment]
Hi. Yes, this must be a useful feature. It is available on many other synths' UIs and some good MIDI players too. Apparently, it was already requested last year at [SourceForge tracker](https://sourceforge.net/p/munt/feature-requests/25/) as well as the ability to [adjust volume per channel](https://sourceforge.net/p/munt/feature-requests/31/). I plan to tackle these having sorted out some more urgent problems, though.

Thanks for the suggestion anyway!

**Comment by sergm** [comment]
For the time being, I'd propose you to have a look at [a good MIDI tool for Windows](http://www.midiox.com/) which I've been used quite extensively.
